source/style-guide/markup/format/headings.txt

```diff
@@ -0,0 +1,128 @@
+=====================
+Headings and Sections
+=====================
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+|rst| uses repeating characters beneath the heading text to set
+headings.
+
+Heading Level Markup
```

Comment:
I think the page could present information more clearly just by showing an example of the recommended way and noting that it still renders correctly under the mentioned circumstance.